src/geff/metadata_schema.py

```diff
@@ -123,6 +124,7 @@ class GeffMetadata(BaseModel):
     )
     directed: bool
     axes: Sequence[Axis] | None = None
+    affine: Affine | None = None
```

Comment:
Can we change this to a `Field(None, description=...)` definition and write the description of how this affine is defined and how it is interpreted? That way we can make sure it makes its way into the schema. It would also be good to update the docs schema example to list an example of the affine field as a concrete thing to refer to.

Reply:
I think in the schema the description is getting autopopulated from the Affine class docstring.

Reply:
Or at the very least I built the docs locally and I'm happy